open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/graph-repository-connector/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/graphrepository/repositoryconnector/GraphOMRSMetadataStore.java

```diff
@@ -146,6 +147,86 @@
     }
 
 
+    /**
+     * Return a list of entities from the store that are at the latest level.
+     *
+     * @return list of EntityDetail objects
+     */
+    synchronized List<EntityDetail>   getEntities()
```

Comment:
I have some concerns about this fix:
- there is no issue explaining why you need this change
- the method seems to be a new package wide visibility method for an implementation of the OMRS, with no-one calling it. It is likely to blow up if there are a large number of entities, which is which we have the find commands to allow you to filter on what you need and page the results. 
- An implementation should not be introducing its own methods , it should be implementing OMRS APIs. In this way the caller can call OMRS and not need to know what the implementation is.

Reply:
There is code maked in local-graph-repository because it does not exist. But in module inmemory-repository - already exist this method. We have some bug with NoMethodException when we migrate from inmemory repository to Janusgraph(Habase) repository

Reply:
> inmemory-repository

Sorry I did not see this comment. There is an unused  getEntities() method in the in memory repository implementation. You should not have to implement this in another connector. I do not see this method in an interface or a base class; but I could have missed something, where are you seeing the NoMethodException - driving the need for this change?

Reply:
Hi, David.
Thank you for reply.
I think that we must try find entityes by properties.
I think this Pull-request do not actual for present time.